package.json

```diff
@@ -38,7 +38,14 @@
     "babel-loader": "^8.0.6",
     "babel-preset-react-app": "^9.1.0",
     "formik": "^2.1.4",
+    "husky": "3.1.0",
+    "pretty-quick": "^2.0.1",
```

Comment:
do we keep the ^ ?

Reply:
Yeah let's keep the `^` for now, but you're right. We should eventually stabilize our package versions.